### DISCUSSION TITLE
Is there a way to typeset .md files (as if they were .tex)?

### DISCUSSION THREAD

**Discussion by kenNash** [James-Yu/LaTeX-Workshop#3302] [Q&A]
Hi all.

I'd really love to be able to enjoy all the link-backlink features of Obsidian.md in my LaTeX documents.

Now:
- Obsidian doesn't do its magic if the file is not a .md one;
- LaTeX-Workshop has a "markdown_latex_combined" mode that allows us to edit a .md file as if it were a .tex.

If only would be possible to typeset such a .md file, the problem would be solved.

Please, please, give me good news! ;-))

**Comment by jlelong** [comment]
Could you be more precise on what you are trying to do?

The `markdown_latex_combined` language is designed to edit markdown code inside a `.tex` file when using the `markdown` package.

**Reply by kenNash** [reply]
Yep, sure!

I would like to edit a file with a .md extension (to benefit some of the features of Obsidian), but with a LaTeX content. Then, I'd like to be able to typeset it, as if it were a .tex file.
